lib/node_modules/@stdlib/stats/base/dsmeanpn/include/stdlib/stats/base/dsmeanpn.h

```diff
@@ -29,9 +29,14 @@ extern "C" {
 #endif
 
 /**
-* Computes the arithmetic mean of a single-precision floating-point strided array using a two-pass error correction algorithm with extended accumulation and returning an extended precision result.
+* Computes the arithmetic mean of a single-precision floating-point strided array `x` using a two-pass error correction algorithm with extended accumulation and returning an extended precision result.
```

Comment:
```suggestion
* Computes the arithmetic mean of a single-precision floating-point strided array using a two-pass error correction algorithm with extended accumulation and returning an extended precision result.
```

I suggest also updating your other PRs accordingly.